src/Maestro/Maestro.Web/Api/v2019_01_16/Controllers/BuildsController.cs

```diff
@@ -90,6 +87,23 @@ public override async Task<IActionResult> GetBuild(int id)
             return Ok(new Build(build));
         }
 
+        [HttpGet("{id}/tree")]
+        [SwaggerApiResponse(HttpStatusCode.OK, Type = typeof(BuildGraph), Description = "The tree of build dependencies")]
+        [ValidateModelState]
+        public async Task<IActionResult> GetBuildGraph(int id)
```

Comment:
Is this always going to return a graph with a large depth because zig-zaggy 'cycles'

Reply:
Are there cycles that don't go through "toolset" legs?

Reply:
It will return them until it hits a depth of 10 if they aren't Toolset dependencies.